src/protocol/bolt/src/main/java/org/apache/jmeter/protocol/bolt/config/BoltConnectionElement.java

```diff
@@ -67,6 +70,7 @@ public void testStarted() {
             synchronized (this) {
                 driver = GraphDatabase.driver(getBoltUri(), AuthTokens.basic(getUsername(), getPassword()));
                 variables.putObject(BOLT_CONNECTION, driver);
+                variables.put(DATABASE, database==null || database.length()==0 ? DEFAULT_DATABASE :database);
```

Comment:
same remark as for `BoltConnectionElementBeanInfo` : should not have a default value here
also: formatting issues (missing spaces)